src/win/pipe.c

```diff
@@ -471,6 +472,8 @@ void uv_pipe_endgame(uv_loop_t* loop, uv_pipe_t* handle) {
 
 
 void uv_pipe_pending_instances(uv_pipe_t* handle, int count) {
+  if (handle->flags & UV_HANDLE_BOUND)
+    return;
```

Comment:
You could consider changing the return type from void to int so you can return UV_EBUSY here.

It's not technically an API break (existing code will just keep on ignoring the return value) and neither is it an ABI break due to how name mangling works in C.